.github/workflows/upgrade-ci.yml

```
@@ -0,0 +1,158 @@
+            exit 0
+          fi
+          export OPM_VERSION=$(curl -s https://api.github.com/repos/operator-framework/operator-registry/releases/latest | grep tag_name | cut -d '"' -f 4)
+
+          # Download the binary
+          curl -Lo opm-linux-amd64 "https://github.com/operator-framework/operator-registry/releases/download/${OPM_VERSION}/linux-amd64-opm"
+          chmod +x opm-linux-amd64
+          sudo mv opm-linux-amd64 /usr/local/bin/opm
+          opm version
+
+      - name: Set Environment Variables
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:54: Double quote to prevent globbing and word splitting

(shellcheck)